src/coreclr/tools/aot/ILCompiler/ILCompiler_publish.csproj

```diff
@@ -2,7 +2,7 @@
   <!-- This project is publishes a self-contained copy of ILCompiler. -->
   <PropertyGroup>
     <_IsPublishing>true</_IsPublishing>
-    <RuntimeIdentifier>$(OutputRID)</RuntimeIdentifier>
+    <RuntimeIdentifier>$(_portableOS)-$(TargetArchitecture)</RuntimeIdentifier>
```

Comment:
I don't like the spread of the underscore-prefixed properties. If we are going to keep spreading it, we should rename the property.

Reply:
We used to have `ToolsRID`, and we were using `MicrosoftNetCoreIlasmPackageRuntimeId` before: https://github.com/dotnet/runtime/commit/891ff91ca2f12c69e619d1035ace6c41a9ff5329#diff-9da24614831c308827a1ae533ffea392c97638c261dd42bd0f5226baa136d16eL252. Was there a reason to remove it?

Reply:
Renamed to PortableOS.